<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="tests" name="Failed suite [Suite]" verbose="0">
  <listeners>
    <listener class-name="rahulshettyacademy.TestComponents.Listeners"/>
  </listeners>
  <test thread-count="5" parallel="tests" name="Submit Order Test(failed)" verbose="0">
    <classes>
      <class name="rahulshettyacademy.Tests.SubmitAloneOrderTest">
        <methods>
          <include name="OrderHistory"/>
          <include name="SubmitAloneOrder" invocation-numbers="0 1"/>
          <include name="tearDown"/>
          <include name="launchApplication"/>
        </methods>
      </class> <!-- rahulshettyacademy.Tests.SubmitAloneOrderTest -->
    </classes>
  </test> <!-- Submit Order Test(failed) -->
  <test thread-count="5" parallel="tests" name="Error Validations Test(failed)" verbose="0">
    <classes>
      <class name="rahulshettyacademy.Tests.ErrorValidationsTest">
        <methods>
          <include name="LoginErrorValidation"/>
          <include name="tearDown"/>
          <include name="launchApplication"/>
          <include name="ProductErrorValidation"/>
        </methods>
      </class> <!-- rahulshettyacademy.Tests.ErrorValidationsTest -->
    </classes>
  </test> <!-- Error Validations Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
